.github/workflows/build-and-test.yml

```diff
@@ -111,13 +111,13 @@ jobs:
 
       # Run tests
       - name: Test Library and Library with coverage
-        run: ./gradlew --build-cache library:test --scan
+        run: ./scripts/build test
 
       - name: Test Harness Tests with coverage
         run: ./gradlew --build-cache testharness:test --scan
```

Comment:
I'm not yet sure what these tests are, but the fail when I try to run them locally.